packages/@aws-cdk/aws-config/lib/rule.ts

```diff
@@ -110,49 +111,46 @@ abstract class RuleNew extends RuleBase {
    */
   public abstract readonly configRuleComplianceType: string;
 
-  protected scope?: CfnConfigRule.ScopeProperty;
+  protected scope?: Scope;
   protected isManaged?: boolean;
   protected isCustomWithChanges?: boolean;
+}
 
-  /**
-   * Restrict scope of changes to a specific resource.
-   *
-   * @see https://docs.aws.amazon.com/config/latest/developerguide/resource-config-reference.html#supported-resources
-   *
-   * @param type the resource type
-   * @param identifier the resource identifier
-   */
-  public scopeToResource(type: string, identifier?: string) {
-    this.scope = {
-      complianceResourceId: identifier,
-      complianceResourceTypes: [type],
-    };
+/**
+ * Determines which resources trigger an evaluation of an AWS Config rule.
+ * @experimental
```

Comment:
What's your thoughts on the graduation process for this? We're doing this as the "last bit" before moving the module/feature to Stable/GA; would these `@experimental` tags be removed at the same time? Are we going to wait a few releases to move to GA to get feedback on this change?

Reply:
I'm okay to drop the `@experimental`. wdyt?